change/react-native-windows-init-2020-07-31-15-32-39-0.63-stable.json

```diff
@@ -0,0 +1,8 @@
+{
+  "type": "none",
```

Comment:
This is a sharp edge, where we will try to bump react-native-windows-init in this branch after we update devDependency and need to specifically add "none" as a change type.